conda_lock/conda_lock.py

```diff
@@ -59,6 +62,38 @@
 CONDA_PKGS_DIRS = None
 DEFAULT_PLATFORMS = ["osx-64", "linux-64", "win-64"]
 
+_PLATFORM_MAP = {"osx-64": "darwin", "linux-64": "linux", "win-64": "win"}
```

Comment:
I wasn't quite sure what other types there are (e,g, `-32` versions). If you provide me with a complete list of what value `platform` in the lockfile can take, I can reflect that here.